### ISSUE TITLE
Rename equinox.bundles repo to equinox

### ISSUE THREAD

**Issue by vogella** [eclipse-equinox/equinox.bundles#18]
PMC decided that we will merge equinox.framework into the equinox.bundles repo. Thomas indicated that a better name would be equinox.

If @tjwatson give his +1 I can open a help ticket to rename the [equinox.bundles](https://github.com/eclipse-equinox/equinox.bundles)
Public repo to equinox.

I did this with GEF Legacy. See https://github.com/eclipse/gef-classic/issues/5

**Comment by vogella** [comment]
We are doing the same for PDE https://github.com/eclipse-pde/eclipse.pde.ui/issues/35

**Comment by vogella** [comment]
@tjwatson please approved. Worked well for pde.ui to pde

**Comment by laeubi** [comment]
@vogella @tjwatson any progress here? Will the merge happen after 2022-06 release?

**Comment by HannesWell** [comment]
> @vogella @tjwatson any progress here? Will the merge happen after 2022-06 release?

That's the plan as far as I know.
Renaming is planned after the merge with equinox.framework (https://github.com/eclipse-equinox/equinox.framework/issues/14) which is just waiting for https://github.com/eclipse-equinox/equinox.framework/pull/44. The latter is waiting for a release of felix-scr: https://github.com/eclipse-equinox/equinox.framework/pull/44#issuecomment-1139207420
But @tjwatson expressed the intention to perform the release soon.

**Comment by laeubi** [comment]
Why is any of those required to merge git repositories?

**Comment by HannesWell** [comment]
It is not required. @vogella probably just wanted to be so kind to avoid the extra work to re-create PRs.
But its statement was from a time where more where open. One PR does not justify a longer delay.
Maybe @tjwatson could give an ETA for the FELIX-SCR release? If that is not too far it would be nice to wait another week or so. :)

**Comment by laeubi** [comment]
It should be possible to simply push the PR to the new remote? But anyways we can wait if there is a ETA and we do not delay it one more release I think.

**Comment by vogella** [comment]
@tjwatson you stay very quit here :-) Can you share you go or no-go for the rename of the repo. We did it in PDE (pde.ui to pde) and IIRC we had no (or at least only little so I forgot them) issues with it.

**Comment by tjwatson** [comment]
Last discussion we had I thought we were leaving both repos as is and creating a new one called `equinox` to merge framework and bundles into.  Then convert the existing repos to read-only with a final commit that removes everything except a readme pointing to the new repo.

**Comment by vogella** [comment]
Sounds good. Shall I open a help desk request to create a new repo or do you want to do this?

**Comment by akurtakov** [comment]
@vogella Please do so. We have to act as we are just spinning here while actual problems are pending and depend (in a way ) on that - https://github.com/eclipse-equinox/equinox.bundles/issues/54#issuecomment-1156018920

**Comment by vogella** [comment]
> @vogella Please do so. We have to act as we are just spinning here while actual problems are pending and depend (in a way ) on that - [#54 (comment)](https://github.com/eclipse-equinox/equinox.bundles/issues/54#issuecomment-1156018920)

https://gitlab.eclipse.org/eclipsefdn/helpdesk/-/issues/1416

**Comment by vogella** [comment]
@akurtakov done, and I pushed content of equinox.bundles to the new equinox repo. https://github.com/eclipse-equinox/equinox 

Can someone update the aggegator build and close all pending PR of equinox-bundles? This process would be less disruptive if we renamed the old repo.

**Comment by laeubi** [comment]
@vogella what about the framework repo? Just in case we adjust the aggregator so no need for double work?

**Comment by vogella** [comment]
> @vogella what about the framework repo? Just in case we adjust the aggregator so no need for double work?

Can you provide a merge PR for it?

**Comment by merks** [comment]
Yes, I've been following this thread and as soon as the framework repo is merged in I can merge the two setups.

**Comment by akurtakov** [comment]
From releng POV I would like to do the aggregator build changes once as we all have an agreement let's just act on it.

**Comment by laeubi** [comment]
Master build is running fine now: https://ci.eclipse.org/equinox/job/equinox/job/master/ so I would assume my part is done here :-)

**Comment by merks** [comment]
@laeubi  So I should look to make the setup work now?

**Comment by laeubi** [comment]
At least everything from both repos is in, so from eclipse ide side we should be fine :+1:
